.travis.yml

```diff
@@ -9,10 +9,12 @@ before_script:
   - bundle exec hatchet ci:setup
 
 script:
-  - docker build --pull --tag heroku-python-build-heroku-16 --file $(pwd)/Dockerfile .
-  - docker run --rm -e "STACK=heroku-16" heroku-python-build-heroku-16 bash $TESTFOLDER
-  - docker build --pull --tag heroku-python-build-heroku-18 --file $(pwd)/Dockerfile.heroku-18 .
-  - docker run --rm -e "STACK=heroku-18" heroku-python-build-heroku-18 bash $TESTFOLDER
+  - docker build --pull --tag travis-build-cedar-14 --file $(pwd)/builds/cedar-14.Dockerfile .
```

Comment:
It's not an issue with this PR, but just documenting my concern that the tests on travis don't use the official images that the buildpack is deployed on

Reply:
Noted. To be clear, the Dockerfiles *do* use the official images:
https://github.com/heroku/heroku-buildpack-python/blob/master/Dockerfile.heroku-18#L1

The dockerfiles don't pull empty ubuntu boxes and try to build from scratch or anything like that. In addition, the previous updates to the heroku-18 dockerfile files ensure that we don't deviate from the official image ( thanks again @dzuelke )

The issue is that the Python buildpack needs Python set up and the Buildpack in the environment, but using docker means we can't use the Python buildpack to install Python or set up the environment. It's an ouroboros problem. See the dockerfiles for more details on what setup is required to build binaries and run tests 👍

Reply:
> the Dockerfiles do use the official images

That is a good point, but it's still a different environment than it's deployed on

> The issue is that the Python buildpack needs Python set up and the Buildpack in the environment, but using docker means we can't use the Python buildpack to install Python or set up the environment. It's an ouroboros problem.

Would using the `build` images fix this? If not, how does python get installed to run the builds in production currently?